lib/python-beta/src/bailo/core/client.py

```diff
@@ -260,6 +258,23 @@ def get_files(
             f"{self.url}/v2/model/{model_id}/files",
         ).json()
 
+    def get_download_file(
```

Comment:
I'm tempted to say this should be special.  Maybe take in a `stream` parameter that you pass to the request and then return the request itself, instead of `request.content`.

That way downstream users can decide whether they want to stream or not.

Reply:
Example: https://stackoverflow.com/questions/16694907/download-large-file-in-python-with-requests